packages/e2e/features/ui/components/storage/storage-browser/upload.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Upload tests within Storage Browser
```

Comment:
`drag-and-drop.feature` renamed to `upload.feature` with two additional tests added